### ISSUE TITLE
Pending transactions disappearing and reposted

### ISSUE THREAD

**Issue by emosto** [ethcore/parity#4177]
Guys, first of all thanks for making parity and providing a great alternative to geth. I have an issue, which previously never had. Making a transaction to a contract never gets registered on etherscan, like it never has been broadcasted. The transactions are staying in "pending" state in the parity signer, forever. Transactions of ether between accounts works perfectly. Anyone had similar issues?

Even more strangely, sometimes - very rarely, the transactions appear on etherscan.io and on next refresh of the browser (to check tx state) the transaction disappears. Then, it appears again (with a new timestamp) and then disappears. I can suppose parity is re-posting the transaction.

Contract creation, TXs for contract functions executions are problematic. Ether transfer not. And, from what I can see in the signer, parity sets the needed gas expenses for the execution of the functions/contract creation. I have confirmed this on parity linux amd64 and windows x64 builds

Parity is fully synced on the live net, no errors in the log.

The calling account has enough ether (5 ether) to do anything it likes up to the gas limit.
gas limit seems not to be the issue since we talk about tx gas of up to 200 000.

I attach screenshots of the behavior as seen from my browser.

Thank you in advance!

![6](https://cloud.githubusercontent.com/assets/16976170/21986629/dfcd6ec6-dc08-11e6-949c-6a25d3fadf12.png)
![5](https://cloud.githubusercontent.com/assets/16976170/21986639/e8fffcca-dc08-11e6-9cff-9f421a3b73ee.png)
![7](https://cloud.githubusercontent.com/assets/16976170/21986643/ee59ab8a-dc08-11e6-8a14-bd5c55f61250.png)
![4](https://cloud.githubusercontent.com/assets/16976170/21986646/f3d6c9e4-dc08-11e6-8153-d8d4f914fb3f.png)

**Comment by tomusdrw** [comment]
tl;dr;
You need to repost previous transaction (set nonce manually) cause it's pending and it's above current gas limit.

Have a look at this transaction:
https://etherscan.io/tx/0x5587a56c5711e502029d5490f43d2a24162240df6dd268de76a2d0e947f74c26
(nonce 17)

It's pending for 2 days already, cause it's slightly above current gas limit. You should replace that transaction before sending any subsequent transactions (have a look at `TxQueue Viewer` dapp included in latest Parity).

Each node has it's own queue and it's own rules regarding when to include transactions (or remove them). Parity is accepting above the gas limit transactions with lowest possible priority (and also when gas limit is within the bounds), so your transaction might sometimes appear in the queue and sometimes not (when gas limit fluctuates), afair geth would reject such transaction right away.

That said, we're working to visualise such situations in GUI more clearly, hopefuly this will help to debug such problems in future.

**Comment by emosto** [comment]
Thank you very much for the detailed info!

**Comment by pyskell** [comment]
To clarify for others you'll need to do the following, if and only if you still want the transaction to be mined. If you want to completely cancel the transaction you'll need to change the `nonce` as previously mentioned and that can't be done through the GUI.

Open `TxQueue Viewer`
![image](https://user-images.githubusercontent.com/9665963/30083726-b038d90a-925d-11e7-8873-fadb591388ac.png)

Find the problem transaction in `TxQueue Viewer`. This is likely going to be one with an inadequate amount of Shannon for the transaction (typically it's 20):
![image](https://user-images.githubusercontent.com/9665963/30083788-daa3661a-925d-11e7-877a-0c6f6d12ff9d.png)

^ The issue, Shannon is set too low

![image](https://user-images.githubusercontent.com/9665963/30083828-f5227c56-925d-11e7-98cc-f79ecb218062.png)

Adjust so that it's a more likely to be mined amount and click Send. Then approve the transaction in the next window that pops up as you normally would.

This new transaction will now replace the old and clear up your queue:

![image](https://user-images.githubusercontent.com/9665963/30083862-16aa9f3e-925e-11e7-8f98-9f949e9ffba2.png)

**Comment by MichaelGoeSlm** [comment]
i'm having exactly the same issue . Can i overwrite the panding transaction using MEW and how should i do it ? Thank you for your help !

**Comment by 84915797613** [comment]
Tôi có tài khoản số dư trong etherscan.io, có bạn nào biết cách giao dịch không,xin liên hệ 0915797613, nếu thành công muốn bao nhiêu eth tôi cũng đưa, cảm ơn

**Comment by Dumiefana** [comment]
![capture](https://user-images.githubusercontent.com/33212452/43459790-3771b134-9501-11e8-9a76-2d18bda7dcd3.PNG)
need help the transaction was sucess..but im still not recive my transfer to westland store.
when i check to txVisualizer no further transaction

**Comment by Dumiefana** [comment]
![capture png 1](https://user-images.githubusercontent.com/33212452/43459985-d5083102-9501-11e8-8312-cf9a81827158.PNG)

**Comment by abunorhA1** [comment]
[77.pdf](https://github.com/paritytech/parity-ethereum/files/3210898/77.pdf)
h   How can I return it to my wallet or re-send it now?
